openupgrade_scripts/scripts/website/15.0.1.0/post-migration.py

```diff
@@ -48,10 +49,67 @@ def update_website_form_call(env):
         view.write({"arch_db": new_arch_db})
 
 
+def update_contact_form_company_description(env):
+    """This script updates the Contact Us form on the website with information from the
+    company model. It retrieves the necessary data such as company name, address, phone,
+    email, and Google Maps link. It then updates the HTML structure of the Contact Us
+    form to display this information."""
+    common_html_block = """
+        <ul class="list-unstyled mb-0 pl-2">
+            <li>%s</li>
+            <li>
+                <i class="fa fa-map-marker fa-fw mr-2"/>
+                <span class="o_force_ltr">%s<br/>
+                &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; %s %s<br/>
+                &amp;nbsp; &amp;nbsp; &amp;nbsp; &amp;nbsp; %s</span>
+            </li>
+            <li>
+                <i class="fa fa-phone fa-fw mr-2"/><span class="o_force_ltr">%s </span>
+            </li>
+            <li><i class="fa fa-1x fa-fw fa-envelope mr-2"/><span>%s</span></li>
+            %s
+        </ul>
+    """
+    for website in env["website"].search([]):
+        website_contactus_view = website.with_context(website_id=website.id).viewref(
+            "website.contactus"
+        )
+        company = website.company_id
+        google_maps_link = (
+            (
+                f'<li><i class="fa fa-1x fa-fw fa-map-marker mr-2"/>'
+                f'<a href="{Markup.escape(company.google_map_link())}" target="_BLANK">'
+                f" Google Maps</a></li>"
+            )
+            if company.google_map_link()
+            else ""
+        )
+        company_info_html = common_html_block % (
+            company.name,
+            company.street,
+            company.city,
+            company.zip,
+            company.country_id.name,
+            company.phone,
+            company.email,
+            google_maps_link,
+        )
+        company_description_pattern = r'<div class="col-lg-4 mt-4 mt-lg-0">(.*?)<\/div>'
+        website_contactus_arch = website_contactus_view.arch_db
+        website_contactus_matches, *_ = re.findall(
```

Comment:
Instead of `findall` + `replace`, you have `sub` method for doing the replacement AFAIK.

Reply:
Yes, that option is also available. I have already made the change, thanks